yarn.lock

```diff
@@ -1,12615 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
For some reason the Yarn lock file has been deleted 😄  Can you re-generate?

Reply:
Yep, fixed